src/browser/components/MattermostView.jsx

```diff
@@ -335,7 +335,7 @@ export default class MattermostView extends React.Component {
     if (this.props.withTab) {
       classNames.push('mattermostView-with-tab');
     }
-    if (!this.props.active || this.state.errorInfo) {
+    if (!this.props.active) {
```

Comment:
not sure of why this change is here :/ seems unrelated to the PR, can you help me understand?

Reply:
See this PR: https://github.com/mattermost/desktop/pull/1127